wiki/Glossary/fr.md

```diff
@@ -0,0 +1,999 @@
+
+### AFK
+
+AFK est un acronyme anglais qui signifie "Away From Keyboard" (littéralement : "loin du clavier").
+Par exemple quand un utilisateur part faire quelque chose d'autre, alors qu'il est encore connecté.
+osu! désigne les joueurs inactifs depuis un certain temps avec la mention "Idle".
+
+### Cercle d'approche
+
+Dans le mode de jeu [osu!standard](/wiki/Game_Modes/osu!/), le cercle d'approche est le cercle extérieur qui entoure le cercle de frappe, le debut d'un slider, ou le centre d'un spinner.
+Quand le cercle d'approche touche la bordure extèrieure du cercle de frappe ou du spinner, vous devez cliquer.
+Pour les spinners, le cercle d'approche indiquera la durée de ce spinner.
+
+### Authentique (terme original : Authentic)
+
+"Authentic" est un terme qui désigne une beatmap qui reprend tous les aspects d'un niveau spécifique des jeux DS originaux (chanson, paramètres, placement des objets, etc.).
+
+## B
+
+### Bancho
+
+Bancho est le serveur "aisément expensible" codé par peppy.
+Il supplante le besoin d'un serveur IRC tiers pour le management du contenu online d'osu!.
+Les joueurs se connectent automatiquement à Bancho quand ils démarrent osu!, et ce si ils sont connectés à internet
+
+### BAT
+
+**Au 1° février 2015, le BAT a été renommé le Beatmap Nomination Group**
+
+BAT&mdash;acronyme pour Beatmap Appreciation Team&mdash;était un groupe d'utilisateurs qui se chargeaient d'approuver et de ranker vos beatmaps .
+
+### Pulsation (terme original : Beat)
+
+La pulsation est un terme qui s'applique à la timeline dans le mode édition.
+La pulsation est une période de temps en lesquelles les chansons peuvent être divisées en fonction de leur BPM.
+Une beatmap bien timée aura toutes les pulsations qui correspondront au rythme de la musique, et une pulsation entière devra tomber entre deux lignes blanches sur la timeline.
+Dépendamment de la division appliquée, d'autres lignes de division de la pulsation peuvent être visibles:
+
+- lignes rouges pour les demi-pulsations,
+- lignes bleues pour les quarts de pulsations,
+- lignes jaunes pour les huitièmes de pulsations, et
+- lignes violettes pour les tiers et sixièmes de pulsations
+
+### Packs de beatmap (terme original : beatmap pack)
+
+Les Beatmap packs _(si le nom est ambïgu pour vous)_ sont des packs de beatmap.
+Les packs de beatmap sont des ensembles de beatmaps rankées et / ou approuvées qui peuvent être téléchargés depuis un service de partage de fichiers (comme [osu!Mirror](https://osu.yas-online.net/) ou [MediaFire](https://www.mediafire.com/)).
+
+La grande majorité des packs sont des ensembles de beatmaps rankées durant la même ppériode de temps. Cependant, certains peuvent contenir des chansons de thème ou de style similaires.
+
+Pour voir la liste des packs, consultez le [Listing des packs de beatmaps](https://osu.ppy.sh/p/packlist)
+
+### Espacement des pulsations (terme original : Beat spacing)
```

Comment:
ditto

if you are going to include the original term, please put it underneath instead

```
### Espacement des pulsations

_Terme original : Beat spacing_
```